modules/base/rendering/screenspacedashboard.cpp

```diff
@@ -40,15 +40,12 @@ namespace {
     constexpr openspace::properties::Property::PropertyInfo UseMainInfo = {
         "UseMainDashboard",
         "Use main dashboard",
-        "If this value is set to 'true', this ScreenSpaceDashboard will use the "
-        "main dashboard instead of creating an independent one.",
+        "If 'true', this ScreenSpaceDashboard will use the main dashboard instead of "
```

Comment:
```suggestion
        "If true, this ScreenSpaceDashboard will use the main dashboard instead of "
```
For consistency with other documentation